estate/views/estate_property_offer_views.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <record id="estate_model_offer_action" model="ir.actions.act_window">
+        <field name="name">Property Offers</field>
+        <field name="res_model">estate.property.offer</field>
+        <field name="view_mode">tree,form</field>
+    </record>
+
+    <record id="estate_model_offer_view_tree" model="ir.ui.view">
+        <field name="name">estate.model.offer.tree</field>
+        <field name="model">estate.property.offer</field>
+        <field name="arch" type="xml">
+            <tree string="Estates">
+                <field name="price"/>
+                <field name="partner_id"/>
+                <field name="status"/>
+            </tree>
+        </field>
+    </record>
+
+    <record id="estate_model_offer_view_form" model="ir.ui.view">
+        <field name="name">estate.model.offer.form</field>
+        <field name="model">estate.property.offer</field>
+        <field name="arch" type="xml">
+            <form string="Estate">
+                <sheet>
+                    <!-- <header>
+                        <h1><field name="name"/></h1>
+                    </header>
+                    <separator/> -->
```

Comment:
Don't push commented code. Once development is done, remove all the commented code or debugging code.